core/src/server/rpc_module.rs

```diff
@@ -598,6 +604,36 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 		Ok(callback)
 	}
 
+	/// Register a new raw RPC method, which computes the response with the given callback.
+	///
+	/// ## Examples
+	///
+	/// ```
+	/// use jsonrpsee_core::server::RpcModule;
+	///
+	/// let mut module = RpcModule::new(());
+	/// module.register_raw_method("say_hello", |_params, _connection_id, _ctx| "lo").unwrap();
+	/// ```
+	pub fn register_raw_method<R, F>(
+		&mut self,
+		method_name: &'static str,
+		callback: F,
+	) -> Result<&mut MethodCallback, RegisterMethodError>
+	where
+		Context: Send + Sync + 'static,
+		R: IntoResponse + 'static,
+		F: Fn(Params, ConnectionId, &Context) -> R + Send + Sync + 'static,
```

Comment:
let's change this to Arc<Context> such that it's Clone + Send + Sync it may be needed for tokio::spawn, tokio::spawn_blocking

```suggestion
		F: Fn(Params, ConnectionId, Arc<Context>) -> R + Send + Sync + 'static,
```

Reply:
That makes sense. I modified this to receive an `Arc<Context>` and added the `Clone` trait bound. I believe we'll have everything we'd need to do a `spawn_blocking`, thanks!